main.tf

```diff
@@ -7,14 +7,14 @@ resource "aws_instance" "webserver" {
   instance_type = "t2.micro"
 
   tags = {
-    Name = "webserver"
+    Name = "webserver1"
   }
 }
 
 resource "aws_s3_bucket" "accuricsbucketdemo" {
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure at-rest server side encryption (SSE) is enabled for AWS S3 Buckets**
 
**Vulnerability**
AWS S3 Buckets do not have at-rest server side encryption enabled to protect sensitive data.
 
**Remediation**
On the AWS S3 console, for each bucket that failed the rule, navigate to the Properties tab, and select Default encryption. Select an encryption method (AES-256 or AWS-KMS).

source: accurics-stageone